local-cli/cliEntry.js

```diff
@@ -137,7 +137,7 @@ function run() {
 
   commander.parse(process.argv);
 
-  const isValidCommand = commands.find(cmd => cmd.name === process.argv[2]);
+  const isValidCommand = commands.find(cmd => cmd.name.split(" ")[0] === process.argv[2]);
```

Comment:
This is needed because the rnpm commands have additional positional arguments, e.g `link <packageName>`, therefore a naive full string comparison wouldn't work.